pants

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env bash
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
This file is copied from https://static.pantsbuild.org/setup/pants
I did not change it at all. Any future updates (which should be infrequent) will just replace this file with a downloaded update. We should not edit this file.

Reply:
I guess you don't see there any risk by these "unattended" updates? I guess it's actually a good way to reduce technical debt and "force" us to stay up-to-date.

Reply:
Updates to pants are not unattended. The version used is pinned in `pants.toml`:
https://github.com/StackStorm/st2/blob/92f08ebafc3882573227ca388f36a9b69e242f89/pants.toml#L2

So, we need a PR + reviews whenever we update the version of pants we're using.

If we need to update this pants launch script, then we also have to manually download + commit the update.

Reply:
Ah ok, that sounds good, thanks!